skyportal/tests/frontend/test_shifts.py

```diff
@@ -51,7 +51,7 @@ def test_super_user_post_shift(
     driver.click_xpath(submit_button_xpath)
 
     # check for dropdown shift
-    driver.wait_for_xpath(f'//span[text()[contains(.,"{form_name}")]]')
+    driver.wait_for_xpath(f'//span[text()[contains(.,"{form_name}")]]', timeout=30)
```

Comment:
This appears to still be flaky.

Reply:
yeah I did some fighting with this stuff on the ExecutedObservations PR, but not sure I have actually resolved this in reality.

Reply:
I'm getting a sense that this is always failing on the dropdown menus. 

Also, maybe mark it as flaky and make a note to come back to it once you figure out the underlying problem (this isn't directly related to this PR).

Reply:
Good idea. Just marked it.